### ISSUE TITLE
How to create the files in data folder for my own dataset

### ISSUE THREAD

**Issue by daydream-boost** [matanhofree/lungTumorEvolution#1]
hi,

We want to plot figures using our own dataset(Seurat Object), 
So how to create the files in data folder for my own dataset?

thanks.

**Comment by erzakiev** [comment]
I agree. Not sure what's the point of putting data on github if no one can use it. Just to tick some boxes?

**Comment by erzakiev** [comment]
@daydream-boost you could have used matlab for that, but the license fee is exhorbitant and (for me at least) the hassle of installing it just for one project isn't worth it
